.ci/package-testing/Jenkinsfile

```diff
@@ -0,0 +1,31 @@
+#!/bin/groovy
+library 'kibana-pipeline-library'
+kibanaLibrary.load()
+kibanaPipeline(timeoutMinutes: 300) {
+  slackNotifications.onFailure(disabled: true) {
+    githubPr.withDefaultPrComments {
```

Comment:
You can just remove this `githubPr.withDefaultPrComments {}` wrapper, since this won't run for PRs

Reply:
Pushed with 2c82a1b